config/environments/development.rb

```diff
@@ -101,4 +101,5 @@
 
   config.tophat_url = 'https://cdn.sit.earthdata.nasa.gov/tophat2/tophat2.js'
 
+  config.cmr_email_frequency = ENV['cmr_email_frequency'].to_i
```

Comment:
this value should default to something reasonable and not require a value to be set. This same comment is for all environments.

In general all configuration should default to something reasonable if possible and allow for OPS to override when needed. In this way configuration files can hold only the required deviations from base making it easy to understand and read or contain settings that must be their like passwords. In general I don't want our config files to be anything like an apache config file